### DISCUSSION TITLE
can i save logs in a queued job with redis driver?

### DISCUSSION THREAD

**Discussion by LordVersA** [spatie/laravel-activitylog#1089] [Q&A]
hi, thanks for your great packages, 
is there any way to save the logs with a queued job in background?
to don't affect web services performance ?
like queued job with redis driver

**Comment by erfsober** [comment]
Hi thanks for your great question, If anyone found a solution please share within this discussion.

**Comment by adityaa-codes** [comment]
hi, did you find the solution for this?

**Reply by LordVersA** [reply]
no i wrote it myself.

**Reply by VinceSanityyy** [reply]
> no i wrote it myself.

can you show me your implementation?

**Comment by VinceSanityyy** [comment]
Hi any updates for this one?
